sys/Makefile.dep

```diff
@@ -1,3 +1,15 @@
+ifneq (,$(filter auto_init_actuators_%,$(USEMODULE)))
```

Comment:
The `auto_init_sensors` module will only provide the function `void auto_init_sensors(void)`, which in absense of submodules is empty. The individual submodules have to pull in the parent `auto_init_sensors` module, as the individual auto init stuff is called from there. (Same approach for actuators.)

So when `auto_init_sensors_sht1x` is used and  no other submodule of `auto_init_sensors`, it should only auto init SHT1x.

It seems to make sense that I adopt another sensor and two actuators to the API (as separate PR for each device) to allow proper testing. But I'd rather wait a bit until the general direction has been agreed upon, in order to avoid rebasing over and over again :-)

Reply:
> So when auto_init_sensors_sht1x is used and no other submodule of auto_init_sensors, it should only auto init SHT1x.

I think this is what I meant. So when you use(module +=) two sensors which provide auto_init functionality and activate auto_init for one of them, there is no stopping the other sensor from beeing auto_inited, right?

Reply:
> So when you use(module +=) two sensors which provide auto_init functionality and activate auto_init for one of them, there is no stopping the other sensor from beeing auto_inited, right?

That would be a bug. If I had sensors foo and bar, and I would use `USEMODULE += foo bar auto_init_sensors_foo` than `auto_init_sensors` should get pulled in as dependency, but not `auto_init_sensors_bar`. That way foo should get initialized, but not bar.

The individual initialization is done in each submodule, e.g. `auto_init_sensors` should in absence of any of its submodule contain no actual initialization code (`void auto_init_sensors(void)` should be empty).

I'm not 100% sure that my implementation is all correct (therefore adapting one more sensor should be done to test this), but it is definitely intended that auto_init for sensors and actuators has do be enabled for each device individually, unless `auto_init_default` is used.